Solutions/Authomize/Analytic Rules/Privileged_Machines_Exposed_to_the_Internet.yaml

```diff
@@ -0,0 +1,62 @@
+id: 72891de4-da70-44e4-9984-35fcea98d000
+name: Privileged Machines Exposed to the Internet
+kind: Scheduled
+description: These are AWS Ec2 machines that are exposed to the internet. You can further filter by tags so that you can, for example, find exposed machines that are also "privileged".
+severity: High
+queryFrequency: 5m
+queryPeriod: 30m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+- Discovery
+- Impact
+relevantTechniques:
+- T1580
+query: |-
+  Authomize_v2_CL
+  | extend EventID = id_s
+  | extend Policy = policy_name_s
+  | extend Severity = severity_s
+  | extend Description = description_s
+  | extend Recommendation = recommendation_s
+  | extend URL = url_s
+  | extend Tactics = tactics_s
+  | where Policy contains "Privileged Machines Exposed to the Internet"
+  | project  EventID, Policy, Severity, Description, Recommendation, URL, Category, Tactics
```

Comment:
Summarize result would be better, since it reduces no. of rows. can we use something like below?
| summarize count() by EventID, Policy, Severity, Description, Recommendation, URL, Category, Tactics

Reply:
also, I would recommend adding binned 'TimeGenerated' column with the summarized result

Reply:
In this specific case - we generally see less than 2 machines shown. We only pull that information once on our side, so it is highly unlikly we will see anything that is too much here. As customers use the integration and we see something different we will update then.